cli/src/swam/cli/Main.scala

```diff
@@ -113,6 +98,13 @@ object Main extends CommandIOApp(name = "swam-cli", header = "Swam from the comm
     }
   }
 
+  val covOpts: Opts[Options] = Opts.subcommand("coverage", "Run a WebAssembly file and generate coverage report") {
+    (mainFun, wat, wasi, time, dirs, trace, traceFile, filter, debug, wasmFile, restArguments, cov, out).mapN {
```

Comment:
@tareq97 why the out parameter is required? it should be optional with "." (current dir) as the default value

The other thing is that if the folder does not exist then it throws an exception. Create the folder if it does not exist

Reply:
OK will remove it.